assets/chezmoi.io/docs/reference/templates/1password-functions/onepasswordResolve.md

```diff
@@ -0,0 +1,20 @@
+# `onepasswordResolve` *url*
```

Comment:
I think that the name should be different and that we can adjust it over time, because this function is (currently) unlike any other 1Password function in that it *only* works with a service account token.

It's also worth noting that the presence of `OP_SERVICE_ACCOUNT_TOKEN` *will* change the behaviour of the 1Password CLI commands, so these two approaches *should not be mixed* unless you’re only working with a service account.

Reply:
Thanks for the review!

How about `onepasswordSDKResolve` is a name?

I'll also make the name of the environment variable containing the token configurable so it can be mixed with the existing 1Password template functions.

Reply:
I think that would work nicely. If I remember reading correctly, the 1Password SDK will automatically read `OP_SERVICE_ACCOUNT_TOKEN` unless a token is provided, but if we always read from `$<onepasswordSDK.token:-${CHEZMOI_OP_SERVICE_ACCOUNT_TOKEN:-${OP_SERVICE_ACCOUNT_TOKEN}}>` then I think that we’re good.